integration-tests/tests/register-user.js

```diff
@@ -17,6 +17,9 @@ const { registerTest } = require('../utils/jest-utils');
 
 const testRegisterUserFn = withUser(async ({ page, testUrl, token }) => {
   await gotoPage(page, testUrl);
+  await findText(page, 'Terra uses cookies');
+  await click(page, clickable({ textContains: 'Agree' }));
+  await delay(500);
```

Comment:
Would you mind adding a comment before the delay?

Reply:
yeah good call

Reply:
The animation this is waiting for is 300ms long, so we might be able to slightly reduce this to 300 or maybe 350.

https://github.com/DataBiosphere/terra-ui/blob/3b2bcf07c1e17dd54a4261af24ad55e6104733e9/src/components/CookieWarning.js#L24

https://github.com/DataBiosphere/terra-ui/blob/3b2bcf07c1e17dd54a4261af24ad55e6104733e9/src/components/CookieWarning.js#L86